dist/openshift/cincinnati.configmap.yaml

```diff
@@ -9,5 +9,5 @@ data:
   gb.repository: "steveej/cincinnati-test"
   gb.log.verbosity: "vvv"
   pe.address: "0.0.0.0"
-  pe.upstream: "http://cincinnati-graph-builder.cincinnati-staging.svc:8181/v1/graph"
+  pe.upstream: "http://cincinnati-graph-builder.cincinnati-staging.svc:8081/v1/graph"
```

Comment:
This should be `8080` since the policy-engine (PE) considers the graph-builder (GB) as its upstream, which runs on `8080`